examples/scripts/SPI_MPI.py

```diff
@@ -0,0 +1,101 @@
+import sys
+import os
+import time
+import numpy as np
+import scipy as sp
+import h5py as h5
+from mpi4py import MPI # module required to use MPI
+import argparse
+
+import pysingfel as ps
+import pysingfel.gpu as pg
+from pysingfel.util import asnumpy, xp
+
+ROOT_DIR = "../.."
```

Comment:
Let's have user define PYSINGFEL_DIR in the environment
ROOT_DIR=os.environ["PYSINGFEL_DIR"]
Then we don't need lines 14-16